docs/configure.asciidoc

```diff
@@ -1,5 +1,28 @@
+////
+I'm confused about the goal of this doc.
+
+With a title like "Configure" I expected to see a list of the
+valid configuration options. Instead it seems to be a list of
+APM Java agent configuration options, and how to update them
+to migrate from the agent to the distro.
+
+Is the intention of this doc to be a migration guide from the APM Java
+agent to the Elastic Distribution for OpenTelemetry Java?
+////
+
 [[configure]]
-== Configure
+== Migrate?
+
+// I'll add this to the automation if we're sure we want this on every page.
+:language: Java
+:language_lc: java
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+:!language:
+:!language_lc:
+:!distro_name:
 
 ////
 This file is auto generated. Please only make changes in `configure.asciidoc.ftl`
```

Comment:
on L31, we say `(the distro)`. On every other page (if we decide to keep this) we say `("the distro")` with quotation marks.